packages/block-library/src/image/edit.native.js

```diff
@@ -722,6 +722,8 @@ export class ImageEdit extends Component {
 			);
 		}
 
+		const isGif = url.endsWith( '.gif' );
```

Comment:
```suggestion
		const isGif = url.includes( '.gif' );
```
I was having trouble getting the banner to display on my test device and figured out it was because the `?w=500` parameter was being added to the end of the image's URL. As there a number of reasons size parameters may get added to the end of the image's URL, I think we should be checking for the inclusion of `.gif` anywhere in the URL, not only at the end. Does that make sense, or are there any issues I'm not seeing with this alternative?

Reply:
I think I'm going to use the suggestion @hypest has instead -- using the image itself instead of the URL since that covers more approaches.

Reply:
@SiobhyB Going to go with the simpler `includes('.gif')` for now since converting image data to binary isn't super simple. Also added a CHANGELOG.md entry.